src/Search/Console.php

```diff
@@ -120,6 +120,10 @@ public function __construct(Application $console, Kernel $app)
         $this->app = $app;
         $this->root = __DIR__.'/../..';
 
+        if (!defined('GEARMAN_INSTALLED')) {
```

Comment:
There is a similar problem with SQS, on one hand it's useful to have things running with the least amount of dependencies and on the other hand it's possible to mask errors instead of discovering them sooner in `ci`, `end2end` and `prod`. Let's keep this for now.